app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -11,53 +10,56 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
-      this.$pageIsNotUsefulButton = $element.find('.js-page-is-not-useful')
-      this.$somethingIsWrongButton = $element.find('.js-something-is-wrong')
-      this.$promptQuestions = $element.find('.js-prompt-questions')
-      this.$promptSuccessMessage = $element.find('.js-prompt-success')
-      this.$somethingIsWrongForm = $(this.somethingIsWrongForm)
-      this.$surveyForm = $(this.surveyForm)
-      this.$surveyWrapper = $element.find('#survey-wrapper')
-
-      var that = this
+      this.pageIsUsefulButton = this.element.querySelector('.js-page-is-useful')
+      this.pageIsNotUsefulButton = this.element.querySelector('.js-page-is-not-useful')
+      this.somethingIsWrongButton = this.element.querySelector('.js-something-is-wrong')
+      this.promptQuestions = this.element.querySelectorAll('.js-prompt-questions')
+      this.promptSuccessMessage = this.element.querySelector('.js-prompt-success')
+      this.surveyWrapper = this.element.querySelector('#survey-wrapper')
+
+      var thisModule = this
       var jshiddenClass = 'js-hidden'
 
       setInitialAriaAttributes()
       setHiddenValues()
 
-      this.$toggleForms.on('click', function (e) {
-        e.preventDefault()
-        toggleForm($(e.target).attr('aria-controls'))
-        trackEvent(getTrackEventParams($(this)))
-        updateAriaAttributes($(this))
-      })
+      for (var j = 0; j < this.toggleForms.length; j++) {
+        this.toggleForms[j].addEventListener('click', function (e) {
+          e.preventDefault()
+          var el = e.target
+          toggleForm(el.getAttribute('aria-controls'))
+          trackEvent(getTrackEventParams(el))
+          updateAriaAttributes(el)
+        })
+      }
 
-      this.$closeForms.on('click', function (e) {
-        e.preventDefault()
-        toggleForm($(e.target).attr('aria-controls'))
-        trackEvent(getTrackEventParams($(this)))
-        setInitialAriaAttributes()
-        revealInitialPrompt()
-        var refocusClass = '.js-' + $(e.target).attr('aria-controls')
-        $element.find(refocusClass).focus()
-      })
+      for (var i = 0; i < this.closeForms.length; i++) {
+        this.closeForms[i].addEventListener('click', function (e) {
+          e.preventDefault()
+          var el = e.target
+          var formToToggle = el.getAttribute('aria-controls')
+          toggleForm(formToToggle)
+          trackEvent(getTrackEventParams(el))
+          setInitialAriaAttributes()
+          revealInitialPrompt()
+          var refocusClass = '.js-' + formToToggle
+          thisModule.element.querySelector(refocusClass).focus()
+        })
+      }
 
-      this.$pageIsUsefulButton.on('click', function (e) {
+      this.pageIsUsefulButton.addEventListener('click', function (e) {
         e.preventDefault()
-        trackEvent(getTrackEventParams(that.$pageIsUsefulButton))
+        trackEvent(getTrackEventParams(thisModule.pageIsUsefulButton))
```

Comment:
Outside the scope of removing jQuery, so just a quesiton - does the use of `trackEvent` mean that the 'Yes' button doesn't do anything if a user doesn't accept website usage cookies or leaves the cookie banner alone?

Reply:
It looks that way. The only thing that button seems to do is fire an event at Google analytics. In theory that shouldn't be visible unless cookies have been consented to, but as you said, outside the scope of removing jQuery.